src/main/ipc.ts

```diff
@@ -210,7 +210,17 @@ export async function init(cwd: string, logHandler: LogHandler) {
     return clipboard.readText()
   })
   ipcMain.handle('electron.clipboard.readImage', () => {
-    return clipboard.readImage().toDataURL()
+    const base64Data = clipboard.readImage().toDataURL()
+
+    // Electron just returns an empty base64 string (for example
+    // 'data:image/png;base64,' when no image was in the clipboard),
+    // we check that here and more conveniently return null instead
```

Comment:
there is also `image.isEmpty()`: https://www.electronjs.org/de/docs/latest/api/native-image#imageisempty

maybe using that results in clearer/prettier code